CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.1.2] - 2019-08-09
```

Comment:
Let's go with 2.2.0 as the new version number as this is more than a bugfix. Can you please update here (you can leave the date as "TBD" and we'll fill it in when we do a release) and also in the pom.xml (change to 2.2.0-SNAPSHOT, we'll set the version when we do a release).

Reply:
@massdosage Thanks for taking a look. I've added a new commit with the version number changes.